activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Add support for supplying `locale` to `transliterate` and `parameterize`
+
+        I18n.backend.store_translations(:de, i18n: { transliterate: { rule: { "ü" => "ue" } } })
+
+        ActiveSupport::Inflector.transliterate("ü", locale: :de) => "ue"
+        "Fünf autos".parameterize(locale: :de) => "fuenf-autos"
+        ActiveSupport::Inflector.parameterize("Fünf autos", locale: :de) => "fuenf-autos"
+
+    *KaanOzkan*
```

Comment:
@KaanOzkan is this the right name?
Also, do I need to add any more contributors?

Reply:
You should definitely add yourself! 
Also `*Kaan Ozkan*` is better. Thank you.

Reply:
Wow, thank you!

I've pushed the changes :)